luzer/tests/test_unit.lua

```diff
@@ -146,14 +146,18 @@ local num = fdp:consume_number(1, MAX_INT)
 assert(type(num) == "number")
 
 -- luzer.FuzzedDataProvider.consume_numbers()
-fdp = luzer.FuzzedDataProvider("ABCDEF")
+fdp = luzer.FuzzedDataProvider(string.rep("XYZ", 100))
 assert(type(fdp.consume_numbers) == "function")
 
-res = fdp:consume_numbers(2, 1, 3)
+local min = 1
+local max = MAX_INT
+res = fdp:consume_numbers(min, max, 3)
```

Comment:
I suppose it is good to check that `#res == 3` as requested.

Reply:
Fixed:

```diff
--- a/luzer/tests/test_unit.lua
+++ b/luzer/tests/test_unit.lua
@@ -151,8 +151,10 @@ assert(type(fdp.consume_numbers) == "function")
 
 local min = 1
 local max = MAX_INT
-res = fdp:consume_numbers(min, max, 3)
+local count = 3
+res = fdp:consume_numbers(min, max, count)
 assert(type(res) == "table")
+assert(#res == count)
 for _, n in ipairs(res) do
     assert(type(n) == "number")
     assert(n <= max)
```